templates/agenda/show.html.twig

```diff
@@ -139,7 +139,7 @@
                                                 </div>
                                                 {% if not agenda.isFinished() %}
                                                 <div class="ml-2">
-                                                    <form id="myform" method="post" action="{{ path('agenda_item_delete', {'id': agenda.id, 'agenda_item_id': agenda_item.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to delete this agenda item?{% endtrans %}');">
+                                                    <form method="post" action="{{ path('agenda_item_delete', {'id': agenda.id, 'agenda_item_id': agenda_item.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to delete this agenda item?{% endtrans %}');">
```

Comment:
See the comment on how to handle the `confirm` message above.

Reply:
Good catch. This is something that will be needed in many other places, so we will fix it in a different branch.